build/README.md

```diff
@@ -48,8 +48,28 @@ Flash the resulting `eos.img` to your SD Card (16GB required, any larger is neit
 
 We recommend [Balena Etcher](https://www.balena.io/etcher/)
 
+Note: The latest version of Balena may not flash `eos.img` properly. Version 1.5.122 should work properly if you are having issues with the latest version of Balena.
+https://github.com/balena-io/etcher/releases/tag/v1.5.122
+
```

Comment:
This can probably just be in the troubleshooting section, hopefully balena have a fix out soon

Reply:
Would you guys like me to move it to the trouble shooting section or nah?

Reply:
We should actually look into whether Balena is aware of the issue

Reply:
Sure thing. I'll pull this out.

Reply:
@kn0wmad removed.